components/mfa/setup/setup.tsx

```diff
@@ -77,6 +80,7 @@ export default class Setup extends React.PureComponent<Props, State> {
             this.props.history.push('/');
             return;
         }
+        trackEvent('mfa_setup', 'mfa_setup_page_loaded', {isMobile: isMobile()});
 
         this.props.actions.generateMfaSecret().then(({data, error}) => {
             if (error) {
```

Comment:
Should failures also be tracked? If yes, the server error id can be a potential property to track.

Reply:
Good idea, added 2 new events to track errors on load and on submit